locale/en/about/working-groups.md

```diff
@@ -3,22 +3,47 @@ layout: about.hbs
 title: Working Groups
 ---
 # Working Groups
+There are 2 types of Working Groups:
+* [Top-Level Working Groups](#top-level-working-groups)
+* [Core Working Groups](#core-working-groups)
 
- Working Groups are autonomous projects created by the
-[Technical Committee (TC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#technical-committee).
+## Top-Level Working Groups
+<!-- Information here should mostly mirror: https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md -->
 
-Working Groups can be formed at any time but must be ratified by the TC.
-Once formed the work defined in the Working Group charter is the
-responsibility of the WG rather than the TC.
+Top-Level Working Groups are created by the 
+[Technical Steering Committee (TSC)](https://github.com/nodejs/TSC#top-level-wgs-and-tlps).
 
-It is important that Working Groups are not formed pre-maturely. Working
-Groups are not formed to *begin* a set of tasks but instead are formed
-once that work is already underway and the contributors
-think it would benefit from being done as an autonomous project.
+### Current Top-Level Working Groups
+* [Inclusivity](#inclusivity)
+
+#### [Inclusivity](https://github.com/nodejs/inclusivity)
+The Inclusivity Working Group seeks to increase inclusivity and diversity for 
+the Node.js project:
+
+* Increasing inclusivity means making the Node.js project a safe and friendly 
+place for people from diverse backgrounds.
+* Increasing diversity means actively onboarding people from diverse backgrounds 
+to the Node.js project and maintaining their participation.
+
+Its responsibilites are:
+* Foster a welcoming environment that ensures participants are valued and can 
+feel confident contributing or joining discussions, regardless of any aspect of 
+their identity.
```

Comment:
I noticed the "including but not limited to" section was left off. The list is an important part of this statement. If we are going to list the responsibilities section then I think it should be listed in full, not summarized.